tests/test_collection_get.py

```
@@ -131,6 +131,313 @@ def _insert_test_data(self, client, collection_name: str):
+            include=["ids"],
+        )
+        assert result and "ids" in result
+        assert set(result["ids"]) == {"id_disjoint", "id_null", "id_empty"}
+
+    def test_embedded_metadata_array_in_overlap(self):
+        """
+        Regression test for JSON array $in operator with embedded client.
+        Includes overlap, disjoint, empty list, and complementary $nin expectations.
+        """
+        try:
+            import pylibseekdb  # noqa: F401
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)